### PR TITLE
Remove trace and normalize trace[X|#] use.

### PR DESCRIPTION
**Pull request by plural**

### CODE REVIEW

File: pack/23s.json [hunk 1]
```diff
@@ -130,9 +130,9 @@
     "position": 7,
     "quantity": 3,
     "side_code": "runner",
-    "stripped_text": "This card is not trashed until another current is played or an agenda is scored. Whenever you steal an agenda, force the Corp to \"Trace 0 If unsuccessful, the Runner gains credits equal to the number of agenda points in both players' score areas.\"",
+    "stripped_text": "This card is not trashed until another current is played or an agenda is scored. Whenever you steal an agenda, force the Corp to \"Trace[0]. If unsuccessful, the Runner gains credits equal to the number of agenda points in both players' score areas.\"",
```
Comment (distributive) [inline pack/23s.json:133]: In this stripped text the trace is formatted with square brackets.

File: pack/23s.json [hunk 2]
```diff
@@ -314,7 +314,7 @@
     "side_code": "corp",
     "stripped_text": "After you resolve this operation, your action phase ends. Play only if the Runner made a run during their last turn. Trace 4. If successful, give the Runner 4 tags.",
```
Comment (distributive) [inline pack/23s.json:315]: In this stripped text the trace is not formatted with brackets.

**Review by distributive — CHANGES_REQUESTED** [review]
The formatting of traces in `stripped_text` fields seems inconsistent.

File: pack/23s.json (hunk 1 above)
Comment (plural) [reply pack/23s.json:133]: yeah, that's intentional.  It should be the case throughout.  this enables x:"trace[3]" searches to do the right thing.

File: pack/23s.json (hunk 2 above)
Comment (plural) [reply pack/23s.json:315]: *this one* was not intentional.  :)

**Review by distributive — APPROVED** [review]
I can't find any more!

**Pull request merged**